src/routes/ui.js

```diff
@@ -17,6 +18,7 @@ if (config.discord.enabled) {
     });
 
     router.get('/logout', (req, res) => {
+        clearOtherSessions(req.session.user_id, '');
         req.session = null;
```

Comment:
Wtf is this line?

Reply:
That logs them out of all sessions. Now that you bring it up, maybe we shouldnt clear all of their sessions? They might only want to log out of the current session they are in?

Reply:
Since we used cookie sessions before, there was no `.destory()` method and you *had* to set the session to null to destroy it. Just an overlook when we converted to mysql sessions.
https://stackoverflow.com/a/49526072

Reply:
Uh

https://github.com/WatWowMap/MapJS/blob/34d2038877ac26fb3a4744385c4ffbaee2440ce2/src/index.js#L85-L93

Reply:
Ok so [`session` is a property in cookie-session](https://github.com/expressjs/cookie-session/blob/8cf0508c2aeae3a649cb4afc9a35b04f7b934c21/index.js#L74). That makes more sense now.

Reply:
Needs to be deleted since we don't use it anymore.